draft-packages/title-block-zen/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@kaizen/draft-zen-title-block",
+  "version": "1.4.5",
```

Comment:
Curious why you're starting the version at 1.4.5 and not 1.0.0. Any particular reason?

Reply:
Ah good catch. I copied this over from the OG Title Block and forgot update that. Will change to 1.0.0.

Reply:
In that case, can you also delete the content of the changelog?